packages/rum-core/src/domain/assembly.spec.ts

```diff
@@ -213,22 +212,6 @@ describe('rum assembly', () => {
           expect(serverRumEvents[0].context).toEqual({ foo: 'bar' })
         })
 
-        it('should reject modification on context field for view events', () => {
```

Comment:
❓ **Question**: I assume this test is removed because it was testing when the FF is disabled 🤔

Reply:
Right. So it was testing the opposite, when the view context is read-only. I kept it while this feature was behind ff to make sure the ff works.